shell/initialize/client.js

```diff
@@ -485,11 +463,6 @@ function fixPrepatch(to, ___) {
     });
 
     checkForErrors(this);
-
-    // Hot reloading
-    if (isDev) {
-      setTimeout(() => hotReloadAPI(this), 100);
-    }
```

Comment:
As mentioned in the PR summary. I noticed that the HMR code wasn't actually being used so I decided to remove it for both fetch and asyncData.

The video I posted demonstrates that HMR was still working in fetch even without this code.